langlib/lang.string/src/main/java/org/ballerinalang/langlib/string/Includes.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.langlib.string;
+
+import io.ballerina.runtime.api.values.BString;
+
+/**
+ * Extern function lang.string:includes(string, string, int).
+ *
+ * @since 2.0.0
+ */
+
+public class Includes {
+    public static boolean includes(BString str, BString substr, int index) {
+        String strS = str.getValue();
+        String substrS = substr.getValue();
+        try {
+            String substring = strS.substring(index);
```

Comment:
This is wrong, Please have a look at this. https://github.com/ballerina-platform/ballerina-lang/blob/0cecff64a0f16d8c181675b2889cb5942c283287/langlib/lang.string/src/main/java/org/ballerinalang/langlib/string/Substring.java#L61